src/App.tsx

```diff
@@ -1,223 +1,145 @@
-                type="checkbox"
-                className="todo__status"
-              />
-            </label>
-
-            <span data-cy="TodoTitle" className="todo__title">Redux</span>
-            <button
-              type="button"
-              className="todo__remove"
-              data-cy="TodoDeleteButton"
-            >
-              ×
-            </button>
-
-            <div data-cy="TodoLoader" className="modal overlay is-active">
-              <div className="modal-background has-background-white-ter" />
-              <div className="loader" />
-            </div>
-          </div>
-        </section>
-
-        <footer className="todoapp__footer" data-cy="Footer">
-          <span className="todo-count" data-cy="todosCounter">
-            4 items left
-          </span>
-
-          <nav className="filter" data-cy="Filter">
-            <a
-              data-cy="FilterLinkAll"
-              href="#/"
-              className="filter__link selected"
-            >
-              All
-            </a>
-
-            <a
-              data-cy="FilterLinkActive"
-              href="#/active"
-              className="filter__link"
-            >
-              Active
-            </a>
-            <a
-              data-cy="FilterLinkCompleted"
-              href="#/completed"
-              className="filter__link"
-            >
-              Completed
-            </a>
-          </nav>
-
-          <button
-            data-cy="ClearCompletedButton"
-            type="button"
-            className="todoapp__clear-completed"
-          >
-            Clear completed
-          </button>
-        </footer>
-      </div>
+        <Header
+          todoTitle={todoTitle}
+          isAdding={isAdding}
+          setTodoTitle={setTodoTitle}
+          handleLoadTodoOnServer={handleLoadTodoOnServer}
+        />
 
-      <div
-        data-cy="ErrorNotification"
-        className="notification is-danger is-light has-text-weight-normal"
-      >
-        <button
-          data-cy="HideErrorButton"
-          type="button"
-          className="delete"
+        <TodoList
+          visibleTodo={visibleTodo}
+          isAdding={isAdding}
+          todoTitle={todoTitle}
+          selectedTodo={selectedTodo}
+          deleteTodoFromServer={deleteTodoFromServer}
         />
 
-        Unable to add a todo
-        <br />
-        Unable to delete a todo
-        <br />
-        Unable to update a todo
+        {todos.length > 0 && (
+          <Footer
+            filteredType={filteredType}
+            notCompletedTodoNumber={notCompletedTodoNumber}
+            completedTodoNumber={completedTodoNumber}
+            setFilteredType={setFilteredType}
+            handleClearCompletedTodos={handleClearCompletedTodos}
+          />
+        )}
       </div>
+
+      <ErrorNotification
+        errorAlert={errorAlert}
+        setIsAdding={setIsAdding}
+        setErrorAlert={setErrorAlert}
+      />
```

Comment:
It will be better to add a condition for rendering this component. You don`t need to render it if you don't have errors at all.